Quentinchampenois/decidim-module-budgets_importer
# fix:unexpected_appearance_in_component

🎩 Description
Fix the unexpected appearance of "budgetimporter" in the list components dropdown 

📌 Related Issues
[Notion card](/opensourcepolitics/Budget-Importer-Le-budget-importer-est-pr-sent-dans-la-liste-des-fonctionnalit-s-des-Process-et-As-6e50351e7a904b4e82945a220b9f5368?pvs=4)

Testing

1. As admin go to dashboard
2. Go to a process 
3. Go to components
4. Click on "add component"
5. You can see that budget_importer isn't displayed anymore 🎉
9. 
10. Option: If you want to confirm that budget importer still works, you can follow this steps => https://github.com/OpenSourcePolitics/decidim-lyon/pull/84


📷 Screenshots
<img width="1441" alt="Capture d’écran 2024-06-25 à 17 41 02" src="https://github.com/OpenSourcePolitics/decidim-module-budgets_importer/assets/143180473/0cb9a357-f54c-4f69-acbf-906dbaefb02a">
